unison-src/transcripts/api-doc-rendering.output.md

```diff
@@ -606,28 +604,11 @@ GET /api/getDefinition?names=term
                                             {
                                                 "contents": [
                                                     {
-                                                        "contents": "`",
-                                                        "tag": "Word"
-                                                    },
-                                                    {
-                                                        "contents": "\"doesn't",
-                                                        "tag": "Word"
-                                                    },
-                                                    {
-                                                        "contents": "typecheck\"",
-                                                        "tag": "Word"
-                                                    },
-                                                    {
-                                                        "contents": "+",
-                                                        "tag": "Word"
-                                                    },
-                                                    {
-                                                        "contents": "1",
-                                                        "tag": "Word"
-                                                    },
-                                                    {
-                                                        "contents": "`",
-                                                        "tag": "Word"
+                                                        "contents": {
+                                                            "contents": "\"doesn't typecheck\" + 1",
+                                                            "tag": "Word"
+                                                        },
+                                                        "tag": "Code"
```

Comment:
is this change intended or ok? (The array length is smaller, with all words in a single element)

Reply:
yeah, it's fine. There was a weird example in the api-docs, it looked like this:

```
`
"doesn't typecheck" + 1
`
```

with a single backtick on each line like that, which would previously parse as the literal backtick.